.eslintrc.cjs

```diff
@@ -49,6 +49,7 @@ module.exports = {
         'import/extensions': 'off',
         'import/named': 'off',
         'import/namespace': 'off',
+        'import/no-named-as-default': 'off',
```

Comment:
This rule change came from **forms-manager**

Sometimes the types are wrong and `default` is actually a named export

E.g. When ES modules are compiled by Babel into CommonJS

```patch
- import XXX from 'xxx'
+ import { default as XXX } from 'xxx'
```